README.md

```diff
@@ -69,16 +69,19 @@ build the internal swagger documentation.
 
 ### Swagger generation
 
-The first step is to generate the swagger documents from the root of the server repository.
+The first step is to generate the swagger documents from the root of the
+[server repository](https://github.com/bitwarden/server), and then copy the resulting `api.json` and
+`api.public.json` to the root of the SDK repository.
```

Comment:
The files that we need are `api.json` and `identity.json`, and there's no need to copy them, the script reads them from the server folder directly as long as the server and sdk folders are at the same directory:

https://github.com/bitwarden/sdk-internal/blob/0a9599e820005b4d608cbc311700ede418f4d671/support/build-api.sh#L10-L11